src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure randomFigureOne = figureSupplier.getRandomFigure();
+        Figure randomFigureTwo = figureSupplier.getRandomFigure();
+        Figure defoltFigure = figureSupplier.getDefoltFigure();
+
+        Figure[] figures = new Figure[] {randomFigureOne, randomFigureTwo, defoltFigure};
+        for (Figure figure : figures) {
```

Comment:
Just make a loop of 6 iterations. In the first half, call getRandomFigure, in the second half, getDefaultFigure